cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -2408,3 +2408,48 @@ func TestWaitReadyForMove(t *testing.T) {
 		})
 	}
 }
+
+func Test_applyMutators(t *testing.T) {
+	tests := []struct {
+		name     string
+		object   client.Object
+		mutators []ResourceMutatorFunc
+		want     *unstructured.Unstructured
+		wantErr  bool
+	}{
+		{
+			name: "do nothing if object is nil",
+		},
+		{
+			name:   "do nothing if mutators is a nil slice",
+			object: test.NewFakeCluster("example", "example").Objs()[0],
+			want: func() *unstructured.Unstructured {
+				g := NewWithT(t)
+				obj := test.NewFakeCluster("example", "example").Objs()[0]
+				u := &unstructured.Unstructured{}
+				to, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+				g.Expect(err).NotTo(HaveOccurred())
+				u.SetUnstructuredContent(to)
+				return u
+			}(),
+		},
+		{
+			name:     "return error if any element in mutators slice is nil ",
```

Comment:
```suggestion
			name:     "return error if any element in mutators slice is nil",
```

Reply:
Fixed in https://github.com/kubernetes-sigs/cluster-api/pull/10951/commits/12b0d07f9962abda86f4549b567c38dd63ab9500